client/connection/openvpn.go

```diff
@@ -19,6 +23,8 @@ func ConfigureVpnClientFactory(
 	openvpnBinary, configDirectory, runtimeDirectory string,
 	signerFactory identity.SignerFactory,
 	statsKeeper bytescount.SessionStatsKeeper,
+	ipResolver ip.Resolver,
```

Comment:
Instead of ipResolver and locationDetector dependencies, it could be simple dependency - detectCountry function.